README.md

```diff
@@ -11,7 +11,7 @@ The only command is "/regen" to regenerate all pending blocks, requires the expl
 # Contributing
 Fork the project.
 Run these commands (use git bash if you're on Windows) (Requires Maven to be properly installed)
-```Bash
+```bash
```

Comment:
Does it make a difference?

Reply:
Semantics, I was in the file fixing other things and didn't like it.

Reply:
I prefer it this way, so please leave it

Reply:
Bash actually isn’t recognized (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). It should be `Shell` or `shell`.